modules/reference/pages/cluster-properties.adoc

```diff
@@ -293,6 +293,50 @@ Max number of simultaneous connections to S3 per shard. Includes connections use
 
 ---
 
+=== cloud_storage_recovery_topic_validation_mode
+
+Validation mode performed before recovering a topic from cloud storage. In case of failure, the reason for failure will appear as ERROR lines in Redpanda log. For each topic, reports errors for all partitions, but for each partition, only the first error is reported.
```

Comment:
```suggestion
Validation mode performed before recovering a topic from cloud storage. In case of failure, the reason for failure will appear as ERROR lines in Redpanda application log. For each topic, reports errors for all partitions, but for each partition, only the first error is reported.
```